src/handlers/completion.rs

```diff
@@ -45,6 +47,7 @@ fn get_imports(
 fn get_ident_name(
     uri: String,
     position: Position,
+    trigger_len: u32,
```

Comment:
I would prefer to add a `get_triggered_ident_name` that accepted the `trigger` and made decision base on that rather than passing around a length.